scripts/error-codes/dev-expression-with-codes.js

```diff
@@ -117,16 +117,6 @@ module.exports = function(babel) {
             if (prodErrorId === undefined) {
               // The error cannot be found in the map.
               node[SEEN_SYMBOL] = true;
-              if (process.env.NODE_ENV !== 'test') {
```

Comment:
Should we still keep this but put it behind some other NODE_ENV? E.g. for npm publish builds.